Update

ebenezergraham opened the pull request:
Created a notification service to utilize the customer adapter, the SMS and email service
Created an authentication class to authenticate with the identity service
Created a JMS listeners
Created the logback file
Refactored the API and Service classes
Refactored the DB migration scripts

ebenezergraham committed:
Refactored the SMSConfiguration,
Created an SMSConfiurationCommand
Created a helper service to connect to the customer service
Removed sample domain models from the service and api

ebenezergraham committed:
Refactored the notification service

	modified:   service/src/main/java/org/apache/fineract/cn/notification/service/NotificationConfiguration.java
	modified:   service/src/main/java/org/apache/fineract/cn/notification/service/listener/CustomerEventListener.java
	modified:   service/src/main/resources/application.yml
	modified:   service/src/main/resources/db/migrations/mariadb/V1__initial_setup.sql
	modified:   service/src/main/resources/db/migrations/mariadb/V2__sms_gateway_configurations.sql
	new file:   service/src/main/resources/db/migrations/mariadb/V3__email_gateway_configurations.sql
	deleted:    .travis.yml

ebenezergraham committed:
update

ebenezergraham committed:
Delete system-analysis.mdj

ebenezergraham committed:
Delete .travis.yml

awasum commented:
@ebenezergraham Add unit tests for your service.
Also ignore these files:
 service/build/
 service/out/
 service/.gradle/
component-test/.gradle/
 api/out/
 api/.gradle

Other devs do not need to see .class files as these can be generated by the IDEs on their machines.

ebenezergraham commented:
Thanks for the feedback. I will rectify this.

On Tue, 14 Aug 2018, 19:14 Awasum Yannick, <notifications@github.com> wrote:

> @ebenezergraham <https://github.com/ebenezergraham> Add unit tests for
> your service.
> Also ignore these files:
> service/build/
> service/out/
> service/.gradle/
> component-test/.gradle/
> api/out/
> api/.gradle
>
> Other devs do not need to see .class files as these can be generated by
> the IDEs on their machines.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/fineract-cn-notifications/pull/1#issuecomment-412907661>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AZ-uanOdMz23LlJ60WHL5NaQR_kW4vTGks5uQulBgaJpZM4V5pZU>
> .
>

ebenezergraham committed:
cleaned up .class files

ebenezergraham committed:
Merge branch 'develop' of https://github.com/ebenezergraham/fineract-cn-notifications into develop

ebenezergraham committed:
removed ignored files

ebenezergraham committed:
Delete PermittableGroupIds.class

ebenezergraham committed:
Moved role and user creation to demo-server

ebenezergraham committed:
Merge branch 'develop' of https://github.com/ebenezergraham/fineract-cn-notifications into develop

ebenezergraham committed:
Created Tests and implement more rest and command operations

Created Component test

Created operations for Client manager, rest controller
Created Tests for sms and email creation and retrieval
Changed javamail version
Corrected inconsistencies in naming conventions
General Bug Fixes
New Domain objects to assist fims-web-app operations
